python/kfserving/kfserving/api/kf_serving_client.py

```diff
@@ -84,36 +87,41 @@ def set_credentials(self, storage_type, namespace=None, credentials_file=None,
             raise RuntimeError("Invalid storage_type: %s, only support GCS, S3 and Azure\
                 currently.\n" % storage_type)
 
-    def create(self, inferenceservice, namespace=None, watch=False, timeout_seconds=600):  # pylint:disable=inconsistent-return-statements
+    def create(self, custom_object, namespace=None, watch=False, timeout_seconds=600, # pylint:disable=inconsistent-return-statements
```

Comment:
Can we have a separate API with create_trained_model ?

Reply:
Within the same class or a whole new class altogether?